packages/cozy-client/src/CozyClient.js

```diff
@@ -62,11 +61,12 @@ class CozyClient {
    * @param  {Array.Link}   options.links  - List of links
    * @param  {Object}       options.schema - Schema description for each doctypes
    */
-  constructor({ link, links, schema = {}, ...options }) {
+  constructor({ link, links, schema = {}, metadata = {}, ...options }) {
```

Comment:
Nit: Should this be called `cozyMetadata`?

Reply:
Probably not, we are speaking of the cozyClient initialization metadata

Reply:
My thinking was that it the data goes into the `cozyMetadata` field. This more or less makes it sound like we're setting the metadata for cozy-client itself, which is not really a thing. But not a big deal.

Reply:
I am not convinced. Let's talk about it tomorow

Reply:
I hesitated but I found it confusing as it will not be injected directly in `cozyMetadata`. It is more data that will allow us to create the `cozyMetadata`. @edas @ptbrowne opinions on this?

Reply:
(sorry about my last comment, I did not refresh before)

Reply:
@edas @ptbrowne @y-lohse what do you think about `appMetadata`?

Reply:
👌 with me

Reply:
👌 with me. It's indeed clearer as appMetadata (we will know that 'version' is the version of the app)

Reply:
Yep, sounds good!